src/components/OrgView/GlobalSettingsButton.js

```diff
@@ -19,24 +19,14 @@ function GlobalSettingsButton({ onOpen }) {
   const [opened, setOpened] = useState(false)
   const containerRef = useRef()
 
-  const handleToggle = useCallback(() => {
-    setOpened(!opened)
-  }, [setOpened, opened])
-  const handleClose = useCallback(() => {
-    // when the popover was open and the user clicked on the button
-    // this handler was being called before the click handler, so the
-    // click handler was re-opening the popover, by having this on the
-    // next tick things happen in order.
-    // Other potential fix: send the event with the onClose handler
-    // and stop propagation, but no event is being sent.
-    setTimeout(() => setOpened(false), 0)
```

Comment:
If you remove this the behaviour the comment was describing will come back

Reply:
This has been fixed in https://github.com/aragon/aragon-ui/pull/472; you can try in the [development build](https://aragon-if0a8ydzq.now.sh/).

Reply:
Niceeeeeee 🎉